programmers/실패율.js

```diff
@@ -0,0 +1,27 @@
+function solution(N, stages) {
+    const result = [];
+    let players = stages.length;
+    
+    for (let i = 0; i < N; i++) {
+        let notClear = stages.filter(p => p === i + 1).length;
+        let failureRate = 0;
+        
+        if (notClear === 0) {
+            failureRate = 0
+        } else {
+            failureRate = notClear / players;
+        }
```

Comment:
```suggestion
       if (!notClear) failureRate = 0
        failureRate = notClear / players;
```
이렇게 하면 줄 수를 조금 더 줄일 수 있을 것 같아요!